src/__configuration__/navigationMenu/navigation.tsx

```diff
@@ -93,8 +95,10 @@ export const pageDefinitions: SimpleNavItem[] = [
             {
                 title: 'Mobile Frameworks',
                 url: 'frameworks-mobile',
+                component: <MarkdownPage title={'Mobile Frameworks Introduction'} markdown={Docs.Development.MobileFrameworks.Intro} />,
```

Comment:
Despite you exported your home.mdx as `Home` in `src/docs/development/frameworks-mobile/index.tsx`, here you are still using `Intro` here